edx_exams/apps/core/models.py

```diff
@@ -16,6 +16,12 @@ class User(AbstractUser):
     """
     full_name = models.CharField(_('Full Name'), max_length=255, blank=True, null=True)
 
+    first_name = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Not sure if first name and last name are needed if full name is already part of the model, @Zacharis278 is there a reason we would want to separate the name?

Reply:
+1 on leaving them out, not least because "first name / last name" is not a universal concept anyway

Reply:
Makes sense, will remove these fields